lib/shopify_app/controller_concerns/frame_ancestors.rb

```diff
@@ -8,7 +8,7 @@ module FrameAncestors
       content_security_policy do |policy|
         policy.frame_ancestors(-> do
           domain_host = current_shopify_domain || "*.#{::ShopifyApp.configuration.myshopify_domain}"
-          "#{ShopifyAPI::Context.host_scheme}://#{domain_host} https://admin.shopify.com"
+          "#{ShopifyAPI::Context.host_scheme}://#{domain_host} https://admin.#{::ShopifyApp.configuration.unified_admin_domain}"
```

Comment:
Should we use the utils method here instead for consistency?
```suggestion
          "#{ShopifyAPI::Context.host_scheme}://#{domain_host} #{ShopifyApp::Utils::unified_admin_domain}"
```

Reply:
Totally missed that it wasn't public 😅 - I am slightly concerned now that having both `unified_admin_path` and `unified_admin_domain` is going to lead to confusion. Maybe keep it this way for now and we think up a way of combining the two methods.